app.js

```diff
@@ -0,0 +1,36 @@
+// Defines an express app that runs the boilerplate codebase.
+
+import express from 'express';
+import fetch from 'node-fetch';
+import path from 'path';
```

Comment:
@ipam73 the list of exceptions in the dashboard was made somewhat arbitrarily when first converting the schools-dashboard to ES6. It looks like this repo is taking the approach of defaulting to no exceptions, which I am in favor of.